app/src/test/java/org/simple/clinic/selectstate/SelectStateUpdateTest.kt

```diff
@@ -47,6 +49,24 @@ class SelectStateUpdateTest {
         ))
   }
 
+  @Test
+  fun `when state is saved, then go to registration screen`() {
+    val states = listOf(
+        TestData.state(displayName = "Andhra Pradesh"),
+        TestData.state(displayName = "Kerala")
+    )
+    val statesLoadedModel = defaultModel
+        .statesLoaded(states)
```

Comment:
We cannot save a state without having selected it, so this model should also have a selected state, right?

Reply:
Ah, I thought I update it. Adding it 👍🏾